### ISSUE TITLE
Integrate with TomTom arrow

### ISSUE THREAD

**Issue by ghost** [teelolws/WhereTheZaralek#6]
Hello,

Can you please add support for TomTom arrow? :)

Thanks

**Comment by teelolws** [comment]
Not really. I don't use and know nothing about TomTom. If you can find someone who does, I can bring in the code.

**Comment by ghost** [comment]
I see, no problem.
TomTom is basically listening to map pins - when rare or event spawns and a map pin happens, the coordinates are simply sent to TomTom to create an arrow and TomTom pin :)

Hopefully someone can help - I dont code but i believe it's relatively simple!
Have a good day

**Comment by Ludovicus-Maior** [comment]
Sorry for the delay.  I was just going through the issues trying to close out stale ones and just noticed this.
I am one of the TomTom maintainers @teelolws .
